upup/pkg/fi/cloudup/scalewaytasks/lb_backend.go

```diff
@@ -163,3 +166,25 @@ func (l *LBBackend) RenderScw(t *scaleway.ScwAPITarget, actual, expected, change
 
 	return nil
 }
+
+type terraformLBBackend struct {
+	LBID            *terraformWriter.Literal `cty:"lb_id"`
+	Name            *string                  `cty:"name"`
+	ForwardProtocol *string                  `cty:"forward_protocol"`
+	ForwardPort     *int32                   `cty:"forward_port"`
+}
+
+func (l *LBBackend) RenderTerraform(t *terraform.TerraformTarget, actual, expected, changes *LBBackend) error {
+	tfName := strings.Replace(fi.ValueOf(expected.LoadBalancer.Name), ".", "-", -1)
+	tf := terraformLBBackend{
+		LBID:            expected.LoadBalancer.TerraformLink(),
+		Name:            expected.Name,
+		ForwardProtocol: expected.ForwardProtocol,
+		ForwardPort:     expected.ForwardPort,
+	}
+	return t.RenderResource("scaleway_lb_backend", tfName, tf)
+}
+
+func (l *LBBackend) TerraformLink() *terraformWriter.Literal {
+	return terraformWriter.LiteralProperty("scaleway_lb_backend", fi.ValueOf(l.Name), "id")
```

Comment:
Need to use same ID as in render.
```suggestion
	tfName := strings.ReplaceAll(fi.ValueOf(expected.LoadBalancer.Name), ".", "-")
	return terraformWriter.LiteralProperty("scaleway_lb_backend", tfName, "id")
```